R/paren_body_linter.R

```diff
@@ -0,0 +1,33 @@
+#' @describeIn linters check that there is a space between the
+#' right parenthesis and the function body when no braces are used
+#' to define a function.
+#'
+#' @export
+paren_body_linter <- function() {
+  Linter(function(source_file) {
+    if (is.null(source_file$xml_parsed_content)) return(NULL)
+    
+    xpath <- paste(
+      "//expr[",
+      "@line1 = preceding-sibling::FUNCTION/@line1",
```

Comment:
Maybe we should also lint `if (x)bla` and `while (x)bla`?

Reply:
Yes, `if`, `for`, `while`, `function`, `\` lambda are the cases where `)` must be followed by a body expression. Am I missing any?

Reply:
I assume for as well.

Reply:
Done

Reply:
Oh, `\` is the new shorthand for `function()`. Added as well.

I can't think about anything else.